src/utils.rs

```diff
@@ -107,4 +137,17 @@ mod test {
             );
         }
     }
+
+    #[test]
+    fn test_normalize_unicode_chars() {
+        const TEST_MAP: [(&str, &str); 3] = [
```

Comment:
nit: you can do
```suggestion
        const TEST_MAP: &[(&str, &str)] = &[
```
so that you don't need to write out the length explicitly. It would make it easier to add / remove test cases.